dockstore-webservice/src/test/resources/notebooks/ipynb/hello.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+  "nbformat": 4,
```

Comment:
Worth doing an example of nbformat 5?

https://nbformat.readthedocs.io/en/latest/changelog.html